libs/partners/upstage/langchain_upstage/embeddings.py

```diff
@@ -48,7 +51,7 @@ class UpstageEmbeddings(BaseModel, Embeddings):
 
     client: Any = Field(default=None, exclude=True)  #: :meta private:
     async_client: Any = Field(default=None, exclude=True)  #: :meta private:
-    model: str = "solar-1-mini-embedding"
+    model: str = "solar-embedding-1-large"
```

Comment:
in general we recommend against setting this default and forcing people to define `UpstageEmbeddings(model="solar-embedding-1-large")` because this is technically a breaking change.

It might be worth shipping that breaking change now and not having to ship more breaking in the future
```suggestion
    model: str
```

Reply:
Thanks. Fixed it.